src/tests/system/tests/test_ldap.py

```diff
@@ -14,178 +14,118 @@
 from sssd_test_framework.topology import KnownTopology
 
 
-@pytest.mark.ticket(bz=[795044, 1695574])
-@pytest.mark.importance("critical")
-@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
-@pytest.mark.parametrize("use_ppolicy", ["true", "false"])
-@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
 @pytest.mark.topology(KnownTopology.LDAP)
-@pytest.mark.require(
-    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
-    "SSSD was built without support for running under non-root",
-)
-@pytest.mark.builtwith("ldap_use_ppolicy")
-def test_ldap__password_change_using_ppolicy(
-    client: Client, ldap: LDAP, modify_mode: str, use_ppolicy: str, sssd_service_user: str
-):
+@pytest.mark.parametrize("modify_mode", ["exop", "ldap_modify", "exop_force"])
+@pytest.mark.importance("critical")
+def test_ldap__ppolicy_user_login_then_changes_password(client: Client, ldap: LDAP, modify_mode: str):
     """
-    :title: Password change using ppolicy
-    :description: PPolicy overlay is the latest implementation of IETF password policy for LDAP.
-    This extends the password policy for the LDAP server and is configured in SSSD using
-    'ldap_use_ppolicy'.
-
-    Two password modification modes are tested, Extended Operation (exop), the default and then
-    LDAP (ldapmodify), set by 'ldap_pwmodify_mode' parameter.
-    :note: This feature is introduced in SSSD 2.10.0
+    :title: User issues a password change after login against ppolicy overlay
+    :description:
+        Password Policy (ppolicy) is a loadable module that enables password policies in LDAP.
+        The feature offers two methods to update the password, external operation (exop) or
+        LDAP modify.
+
+        The 'test_authentication__change_password' test is a generic provider test that already
+         covers  LDAP. This test is an edited copy that only tests LDAP with the ppolicy overlay.
     :setup:
-        1. Add a user to LDAP
-        2. Configure the LDAP ACI to permit user password changes
-        3. Set "ldap_pwmodify_mode"
-        4. Start SSSD
+        1. Create user 'user'
```

Comment:
Were we going to suggest dropping specifics like 'user' from the docstring setup and steps?